example_run.py

```
@@ -0,0 +1,26 @@
+from datetime import date
+from tqsdk import TqApi, TqSim, TqAccount, TqBacktest, TqAuth
+
+def run_backtest():
+    api = TqApi(backtest=TqBacktest(start_dt=date(2021, 1, 1), end_dt=date(2021, 12, 31)), web_gui=True, auth=TqAuth("13760685574", "xdy19870920"))
```

Ruff (0.8.2)

Local variable `api` is assigned to but never used

Remove assignment to unused variable `api`

(F841)